main.cpp

```diff
@@ -111,6 +109,18 @@ void CodingLanguage::compileCode(float code)
     compileDCode = code * numOfThingsACompilerDoes;
 }
 
+void CodingLanguage::addTwo(float initialValue, int timesToAddTwo)
+{
+    std::cout << "Starting value: " << initialValue << ". ";
+    
+    for (int i = 0; i < timesToAddTwo; i++)
```

Comment:
Always prefer pre-increment, instead of post-increment.
if you want to know why, ask in Slack.
- [ ] revise